### PR TITLE
SOP for data submitters to upload data files

### PR DESCRIPTION
**Pull request by malloryfreeberg**
This PR requests integrating into the master branch an SOP document for data contributors to upload data files to an upload area. Data wranglers would like to be able to link to these instructions when interacting with new data contributors.

### CODE REVIEW

File: docs/Pilot_submitters_SOP.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+A list of files present in the selected upload area can be viewed by running:
+
+```
+hca upload list -l
+```
+
+## Note
+
+* Files cannot be deleted from the upload area using the `hca` tool. If there are files that need to be deleted, please contact data-help@humancellatlas.org and include your name, credentials, and a list of filenames to delete.
+* Uploading a file with the same filename as one that already exists in the upload area will replace the file with the most recent upload. You can find out what files are already in the system using the command hca upload list -l
```
Comment (sampierson) [inline docs/Pilot_submitters_SOP.md]: Put `hca upload list -l` in backquotes.

Comment (malloryfreeberg) [reply docs/Pilot_submitters_SOP.md]: Done!

**Comment by malloryfreeberg** [comment]
I have rebased to master and now all the checks have passed. I'm not sure what your policy is on merging one's own PRs. Happy to do it myself, or happy for anyone else to do it! Also @hewgreen could you approve this PR if you're happy with it?

**Comment by Bento007** [comment]
@malloryfreeberg, you need one approval. You may squash and merge your changes into master.

**Pull request merged**